opencga-app/app/scripts/azure/arm/azurebatch/pool.json

```diff
@@ -0,0 +1,48 @@
+{
+    "id": "POOL_ID_HERE",
+    "vmSize": "VM_SIZE_HERE",
+    "enableAutoScale": true,
+    "autoScaleFormula": "startingNumberOfVMs = 1;maxNumberofVMs = 3;pendingTaskSamplePercent = $PendingTasks.GetSamplePercent(160 * TimeInterval_Second);pendingTaskSamples = pendingTaskSamplePercent < 70 ? startingNumberOfVMs : avg($PendingTasks.GetSample(160 * TimeInterval_Second));$TargetDedicatedNodes=min(maxNumberofVMs, pendingTaskSamples);",
```

Comment:
Could this formula be parameterized in the script if needed? Let the user set the scale limits.

Reply:
Good spot, that was something that got lost in my move from ARM -> ACI ... will fix that up as have a param for `maxNodeCount`